python/foglamp/plugins/south/cc2650async/cc2650async.py

```diff
@@ -0,0 +1,354 @@
+# -*- coding: utf-8 -*-
```

Comment:
no need to make a new package;  Instead make `south/cc2650poll` > `south/cc2650` and keep existing `__init__.py` and move this `cc2650async.py` to that package. `cc2650poll.py` already exists.

Update README.rst with 2 sections and common class its using

Reply:
Current approach requires module name and folder name to be same. Hence we need to keep two different folder.